docs-chef-io/content/inspec/glossary.md

```diff
@@ -113,7 +113,7 @@ _'Cadillac'_ is an [expected result](#expected-result). Some matchers take an ex
 
 #### its('count') { should _be >=_ 10 }
 
-_be >=_ is an [operator matcher](#operator matcher). It allows you to perform numeric comparisons. All plural resources have a `count` property.
+_be >=_ is an [operator matcher](#operator-matcher). It allows you to perform numeric comparisons. All plural resources have a `count` property.
```

Comment:
There's no Operator Matcher heading/section. Should I yank the link or add a "Operator Matcher" heading and section?

Reply:
It looks like those only work for `be`. https://docs.chef.io/inspec/matchers/#be. Use that link?

Reply:
I just decided to add the section.